frontend/src/components/TicketCard.tsx

```diff
@@ -0,0 +1,182 @@
+});
+
+const Content = styled(CardContent)({
+  padding: '38px',
+  paddingBottom: '0px',
+});
+
+const BusInfo = styled(Box)({
+  backgroundColor: '#FFC107',
+  padding: '8px 16px',
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  borderBottomLeftRadius: '16px',
+  borderBottomRightRadius: '16px',
+});
+
+const BusNumber = styled(Typography)({
+  fontSize: '1.75rem',
+  fontWeight: 'bold',
+  color: '#fff',
+});
+
+const TimeDate = styled(Box)({
+  textAlign: 'right',
+  color: '#fff',
+});
+
+const DashedLine = styled(Box)({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  margin: '16px 0',
+});
+
+const DashedSegment = styled(Box)({
+  flex: 1,
+  borderTop: '2px solid #FFC107',
+  height: '4px',
+  margin: '0 5px',
+});
+
+const Circle = styled(Box)({
+  width: '14px',
+  height: '14px',
+  borderRadius: '50%',
+  backgroundColor: '#FFC107',
+});
+
+const LocationInfo = styled(Box)({
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  paddingBottom: '8px',
+});
+
+const Location = styled(Box)({
+  display: 'flex',
+  flexDirection: 'column',
+});
+
+const BoldText = styled(Typography)({
+  fontWeight: 'bold',
+  WebkitTextStrokeWidth: '0.5px',
+});
+
+const BoldFooterText = styled(Typography)({
+  fontWeight: 'bold',
+  fontSize: '1.25rem',
+});
+
+interface TicketCardProps {
+  busNumber: string;
+  departure: string;
+  destination: string;
+  time: string;
+  date: string;
+  ticketId: string;
+}
+
+const TicketCard: React.FC<TicketCardProps> = ({
+  busNumber,
+  departure,
+  destination,
+  time,
+  date,
+  ticketId,
+}) => {
+  return (
+    <Root>
+      <Header>
+        <HeaderText>{ticketId}</HeaderText>
+      </Header>
+      <Content>
+        <LocationInfo>
+          <Location>
+            <BoldText variant="h6">{departure}</BoldText>
+            <BoldText variant="subtitle2" color="textSecondary">
+              PDPM IIITDM Jabalpur
+            </BoldText>
+          </Location>
+          <Location style={{textAlign: 'right'}}>
+            <BoldText variant="h6" align="right">
+              {destination}
+            </BoldText>
+            <BoldText variant="subtitle2" color="textSecondary" align="right">
+              Jabalpur City
+            </BoldText>
+          </Location>
+        </LocationInfo>
+        <DashedLine>
+          <Circle />
+          <DashedSegment />
+          <DashedSegment />
+          <DashedSegment />
+          <DashedSegment />
+          <DashedSegment />
+          <DashedSegment />
+          <DashedSegment />
+          <DashedSegment />
+          <DashedSegment />
+          <DashedSegment />
+          <DashedSegment />
+          <DashedSegment />
+          <Circle />
+        </DashedLine>
+      </Content>
+      <BusInfo>
+        <Box>
+          <BoldFooterText variant="subtitle2" style={{color: '#fff'}}>
+            Bus No.
+          </BoldFooterText>
```

Comment:
This component `fontWeight `& `fontSize `is supposed to be less than current as per designs.